release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -527,6 +527,11 @@ With this release, {product-title} introduces the Data Processing Unit (DPU) Ope
 
 Administrators can now use the `ClusterUserDefinedNetwork` custom resource to deploy secondary networks on a `Localnet` topology. This feature allows pods and virtual machines connected to the localnet network to egress to the physical network. For more information, see xref:../networking/multiple_networks/primary_networks/about-user-defined-networks.adoc#nw-cudn-localnet[Creating a ClusterUserDefinedNetwork CR for a Localnet topology].
 
+[id="ocp-4-19-whereabouts-ipam_{context}"]
+==== Fast IPAM configuration for the Whereabouts IPAM CNI plugin (Technology Preview)
+
+To improve the performance of Whereabouts, especially if nodes in your cluster run a high amount of pods, you can now enable the Fast IP Address Management (IPAM) feature. The Fast IPAM feature uses `nodeslicepools`, which are managed by the Whereabouts Controller, to optimize IP allocation for nodes. For more information, see xref:../networking/multiple_networks/secondary_networks/configuring-ip-secondary-nwt.adoc#nw-multus-whereabouts-fast-ipam_configuring-additional-network[Fast IPAM configuration for the Whereabouts IPAM CNI plugin].
```

Comment:
Does the '...IP allocation...' refer to IP address allocation? If so, change to 'IP address allocation', or for a generic reference, what you have is fine.
SSG: https://redhat-documentation.github.io/supplementary-style-guide/#_i